frontend/src/views/main/project/TestSuiteExecutionHeader.vue

```diff
@@ -1,124 +1,127 @@
-                        No execution has been performed yet!
-                    </h4>
-                    <h4 v-else-if="props.execution.result === TestResult.ERROR" class="text-alert d-flex">
-                        An error occurred during the execution. Executed <strong class="ml-2">{{
-                            timeSince(execution.executionDate)
-                        }}</strong><v-spacer/>
-                      <v-btn color="error" small @click="openLogs">execution logs.</v-btn>
-                    </h4>
-                    <h4 v-else class="text-alert d-flex">Test suite
-                        {{ props.execution.result === TestResult.PASSED ? 'passed' : 'failed' }}:
-                        <span v-if="successRatio.failed > 0">{{ plurialize('test', successRatio.failed) }} failed</span>
-                        <span v-if="successRatio.failed > 0 && successRatio.passed > 0">, </span>
-                        <span v-if="successRatio.passed > 0">{{ plurialize('test', successRatio.passed) }} passed</span>
-                        <span v-if="successRatio.failed > 0 || successRatio.passed > 0">. </span>
-                        Executed
-                        <span class="ml-2">{{ timeSince(execution.executionDate) }}</span>
-                        <v-spacer/>
-                        <v-btn :color="props.execution?.result === TestResult.PASSED ? 'primary' : 'error'" small @click="openLogs">execution logs</v-btn>
-                    </h4>
-                </v-col>
-            </v-row>
-        </v-alert>
-    </div>
+  <div class='d-flex w100 align-start'>
+    <v-alert :color='testResultStyle.color' :icon='testResultStyle.icon' class='flex-grow-1' prominent
+             text>
+      <v-row align='center'>
+        <v-col class='grow'>
+          <h4 v-if='!props.execution' class='text-alert'>
+            No execution has been performed yet!
+          </h4>
+          <h4 v-else-if='props.execution.result === TestResult.ERROR' class='text-alert d-flex'>
+            An error occurred during the execution. Executed <strong class='ml-2'>{{
+              timeSince(execution.executionDate)
+            }}</strong>
+            <v-spacer />
+            <v-btn color='error' small @click='openLogs'>execution logs.</v-btn>
+          </h4>
+          <h4 v-else class='text-alert d-flex flex-wrap'>Test suite
+            {{ props.execution.result === TestResult.PASSED ? 'passed' : 'failed' }}:
+            <span v-if='successRatio.failed > 0'>{{ plurialize('test', successRatio.failed) }} failed</span>
+            <span v-if='successRatio.failed > 0 && successRatio.passed > 0'>, </span>
+            <span v-if='successRatio.passed > 0'>{{ plurialize('test', successRatio.passed) }} passed</span>
+            <span v-if='successRatio.failed > 0 || successRatio.passed > 0'>. </span>
+            Executed
+            <span class='ml-2'>{{ timeSince(execution.executionDate) }}</span>
```

Comment:
I think we should add some spaces here.
```suggestion
            <span v-if='successRatio.failed > 0'>&nbsp;{{ plurialize('test', successRatio.failed) }} failed</span>
            <span v-if='successRatio.failed > 0 && successRatio.passed > 0'>, </span>
            <span v-if='successRatio.passed > 0'>&nbsp;{{ plurialize('test', successRatio.passed) }} passed</span>
            <span v-if='successRatio.failed > 0 || successRatio.passed > 0'>. </span>
            &nbsp;Executed
            <span>&nbsp;{{ timeSince(execution.executionDate) }}</span>
```

Reply:
Instead of space I removed the parent flex that was not required, the space are not needed anymore